app/models/InputForms.scala

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
There is some interesting stuff here. Other services I have worked on tend to have a separate 'forms" packages for controller forms.

Reply:
They always map to a case class anyway so I like them inside the companion object myself though nobody else seems to do that!

Reply:
Have to concur with Martin here. Much like JSON Formats and other type classes, a `Form` defines behaviour for a particularly class and, as such, belongs to that class's companion object.